config/filter.d/sshd.conf

```diff
@@ -33,6 +33,7 @@ failregex = ^%(__prefix_line)s(?:error: PAM: )?[aA]uthentication (?:failure|erro
             ^(?P<__prefix>%(__prefix_line)s)User .+ not allowed because account is locked<SKIPLINES>(?P=__prefix)(?:error: )?Received disconnect from <HOST>: 11: .+ \[preauth\]$
             ^(?P<__prefix>%(__prefix_line)s)Disconnecting: Too many authentication failures for .+? \[preauth\]<SKIPLINES>(?P=__prefix)(?:error: )?Connection closed by <HOST> \[preauth\]$
             ^(?P<__prefix>%(__prefix_line)s)Connection from <HOST> port \d+(?: on \S+ port \d+)?<SKIPLINES>(?P=__prefix)Disconnecting: Too many authentication failures for .+? \[preauth\]$
+             ^%(__prefix_line)spam_unix\(sshd:auth\):\s+authentication failure.*rhost=<HOST>\s.*$
```

Comment:
There is a problem here that we have ```.*``` on both sides of the ```<HOST>``` and after HOST there is a "user" entry, which would be what user provides... and since first .* is greedy one I am afraid that via "user" field he would be able to inject more than necessary. But anchoring at the end also might not be as efficient so I see only that you need to enter all those previous fields before rhost:
```;\s*logname=\S*\s*uid=\d*\s*euid=\d*\s*tty=\S*\s*ruser=<HOST>.*$```

please try